modules/kmm-building-in-cluster.adoc

```diff
@@ -45,3 +45,5 @@ Otherwise, KMM creates a `Build` resource to build your image. After the image i
 <6> Required.
 <7> Optional: Avoid using this parameter. If set to `true`, KMM will be allowed to check if the container image already exists using plain HTTP.
 <8> Optional: Avoid using this parameter. If set to `true`, KMM will skip any TLS server certificate validation when checking if the container image already exists.
+
+Successful build pods are garbage-collected immediately, unless `job.gcDelay` is set in the Operator configuration. Failed build pods are always preserved and must be deleted manually by the administrator for the build to be restarted.
```

Comment:
I don't _think_ garbage collected should be hyphenated (Googled and didn't see hyphenated much). 

```suggestion
Successful build pods are garbage collected immediately, unless the `job.gcDelay` parameter is set in the Operator configuration. Failed build pods are always preserved and must be deleted manually by the administrator for the build to be restarted.
```

Reply:
Made these changes.